docs/resources/built-in-rules.md

```diff
@@ -330,6 +330,25 @@ Verifies that each tag has a description.
 
 Verifies that tags (names) are declared in alphabetical order.
 
+### response-contains-header
+
+Enforces using specified response headers.
+
+```yaml
+lint:
+  rules:
+    response-contains-header:
+      severity: error
+      names:
+        2xx: 
+          - x-request-id
+        400:
+          - Content-Length
+```
+
+### response-contains-property
+
+The same as `response-contains-header` except for a response properties.
```

Comment:
Let's define it inline.
```suggestion
Enforces definition of specific response properties based on HTTP status code or HTTP status code range.
Priority is given to more precise status codes over the status code range.
```yaml
response-contains-header:
      severity: error
      names:
        2XX: 
          - created_at
        400:
          - title
```
```